globals.go

```diff
@@ -98,6 +98,10 @@ const (
 
 	PushStage = "push"
 
+	MergeTestStarted   = "started"
+	MergeTestSucceeded = "succeeded"
+	MergeTestFailed    = "failed"
```

Comment:
Test seemed appropriate because these statuses are used when we create the patch and when the entire patch is complete, as opposed to for the individual task.